sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/BaseClientBuilder.java

```diff
@@ -25,6 +25,7 @@
 import com.azure.storage.common.policy.RequestRetryOptions;
 import com.azure.storage.common.policy.RequestRetryPolicy;
 import com.azure.storage.common.policy.ResponseValidationPolicyBuilder;
+import com.azure.storage.common.policy.ScrubEtagPolicy;
```

Comment:
This builder is going to be removed as an FYI.

Reply:
OK. I will resolve this after the PR merged in.